app-text/podofo/podofo-0.10.1.ebuild

```
@@ -52,5 +55,23 @@ src_configure() {
 		$(cmake_use_find_package png PNG)
 		$(cmake_use_find_package fontconfig Fontconfig)
 	)
+
+	if test-flags-CXX -ffp-contract=off; then
+		# some optimizations cause testsuite failures which may indicate
+		# unsoundness with contraction. Be cautious for now. Reported
+		# upstream as https://github.com/podofo/podofo/issues/103
+		append-cxxflags -ffp-contract=off
```

This is fine, but just as a note, one can flatten this into:
```suggestion
		append-cxxflags $(test-flags-CXX -ffp-contract=off)
```

---

If we reckon that there's a risk of it being a runtime problem, we need a new revision for the -ffp-contract change.

---

In my dream world, they would not be able to build podofo since they would run with FEATURES=test.